From e41ee95c27e8ba7493aaba5ef2bcaae298175277 Mon Sep 17 00:00:00 2001
From: Linu Cherian <lcherian@marvell.com>
Date: Wed, 12 Sep 2018 12:29:53 +0530
Subject: [PATCH 0187/1051] net: octeontx2: Fix NIX_LF_START/STOP for LBK VFs

Remove CGX channel checks in otx2_rxtx_enable which
is no longer valid. This enables NIX_LF_START/STOP requests
for LBK VFs.

Signed-off-by: Linu Cherian <lcherian@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/otx2_txrx.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/otx2_txrx.c b/drivers/net/ethernet/marvell/octeontx2/otx2_txrx.c
index 7e5819aa14d3..cdfc8aea760d 100644
--- a/drivers/net/ethernet/marvell/octeontx2/otx2_txrx.c
+++ b/drivers/net/ethernet/marvell/octeontx2/otx2_txrx.c
@@ -557,9 +557,6 @@ int otx2_rxtx_enable(struct otx2_nic *pfvf, bool enable)
 {
 	struct msg_req *msg;
 
-	if (pfvf->tx_chan_base < CGX_CHAN_BASE)
-		return 0;
-
 	if (enable)
 		msg = otx2_mbox_alloc_msg_NIX_LF_START_RX(&pfvf->mbox);
 	else
-- 
2.17.1

